### PR TITLE
[DOCS] Added Documentation of the rest of the AudioStream files

### PR DESCRIPTION
**Pull request by Grosskopf**
weird, somehow a collision happened

### CODE REVIEW

File: doc/classes/AudioStreamOGGVorbis.xml [hunk 1]
```diff
@@ -56,10 +56,13 @@
 	</methods>
 	<members>
 		<member name="data" type="PoolByteArray" setter="set_data" getter="get_data">
+			Raw audio data.
 		</member>
 		<member name="loop" type="bool" setter="set_loop" getter="has_loop">
+			If [code]'true'[/code], audio is looping.
```
Comment (cbscribe) [inline doc/classes/AudioStreamOGGVorbis.xml]: "If [code]'true'[/code], audio will loop continuouslyl.
Add default value.

File: doc/classes/AudioStreamRandomPitch.xml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamRandomPitch" inherits="AudioStream" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Plays audio with random pitch tweaking.
 	</brief_description>
 	<description>
+		is used to make playback audio a bit more random.
```
Comment (cbscribe) [inline doc/classes/AudioStreamRandomPitch.xml]: This is a very vague description. Something more like: "Randomly varies pitch during playback."

File: doc/classes/AudioStreamRandomPitch.xml [hunk 3]
```diff
@@ -40,8 +42,10 @@
 	</methods>
 	<members>
 		<member name="audio_stream" type="AudioStream" setter="set_audio_stream" getter="get_audio_stream">
+			The current [audio_stream].
```
Comment (cbscribe) [inline doc/classes/AudioStreamRandomPitch.xml]: Bad link: [AudioStream]

File: doc/classes/AudioStreamRandomPitch.xml [hunk 4]
```diff
@@ -40,8 +42,10 @@
...
 		</member>
 		<member name="random_pitch" type="float" setter="set_random_pitch" getter="get_random_pitch">
+			The higher this value, the more extreme the pitch changes from source randomly.
```
Comment (cbscribe) [inline doc/classes/AudioStreamRandomPitch.xml]: The intensity of random pitch variation.

File: doc/classes/AudioStreamSample.xml [hunk 5]
```diff
@@ -110,20 +112,45 @@
 	</methods>
 	<members>
 		<member name="data" type="PoolByteArray" setter="set_data" getter="get_data">
+			Raw audio data.
 		</member>
 		<member name="format" type="int" setter="set_format" getter="get_format" enum="AudioStreamSample.Format">
+			Sets the audio format to [FORMAT_8_BITS], [FORMAT_16_BITS], [FORMAT_IMA_ADPCM].
 		</member>
 		<member name="loop_begin" type="int" setter="set_loop_begin" getter="get_loop_begin">
+			Sets loop begin in bytes.
```
Comment (cbscribe) [inline doc/classes/AudioStreamSample.xml]: Don't use "sets" - the member variable is a setter *and* a getter.

"The loop start point, in bytes.

File: doc/classes/AudioStreamSample.xml [hunk 6]
```diff
@@ -110,20 +112,45 @@
 	</methods>
 	<members>
 		<member name="data" type="PoolByteArray" setter="set_data" getter="get_data">
+			Raw audio data.
 		</member>
 		<member name="format" type="int" setter="set_format" getter="get_format" enum="AudioStreamSample.Format">
+			Sets the audio format to [FORMAT_8_BITS], [FORMAT_16_BITS], [FORMAT_IMA_ADPCM].
```
Comment (cbscribe) [inline doc/classes/AudioStreamSample.xml]: Don't use "sets" - the member variable is a setter and a getter.

Audio format. See FORMAT_* constants for values.

File: doc/classes/AudioStreamSample.xml [hunk 7]
```diff
@@ -110,20 +112,45 @@
...
 		</member>
 		<member name="loop_begin" type="int" setter="set_loop_begin" getter="get_loop_begin">
+			Sets loop begin in bytes.
 		</member>
 		<member name="loop_end" type="int" setter="set_loop_end" getter="get_loop_end">
+			Sets loop end in bytes.
 		</member>
 		<member name="loop_mode" type="int" setter="set_loop_mode" getter="get_loop_mode" enum="AudioStreamSample.LoopMode">
+			Decides how audio loops, could be [LOOP_DISABLED], [LOOP_FORWARD], [LOOP_PING_PONG].
```
Comment (cbscribe) [inline doc/classes/AudioStreamSample.xml]: Loop mode.  See LOOP_* constants for values.

File: doc/classes/AudioStreamSample.xml [hunk 8]
```diff
@@ -110,20 +112,45 @@
...
+			Raw audio data.
 		</member>
 		<member name="format" type="int" setter="set_format" getter="get_format" enum="AudioStreamSample.Format">
+			Sets the audio format to [FORMAT_8_BITS], [FORMAT_16_BITS], [FORMAT_IMA_ADPCM].
 		</member>
 		<member name="loop_begin" type="int" setter="set_loop_begin" getter="get_loop_begin">
+			Sets loop begin in bytes.
 		</member>
 		<member name="loop_end" type="int" setter="set_loop_end" getter="get_loop_end">
+			Sets loop end in bytes.
```
Comment (cbscribe) [inline doc/classes/AudioStreamSample.xml]: Don't use "sets" - the member variable is a setter and a getter.

Loop end point, in bytes.

**Review by cbscribe — CHANGES_REQUESTED** [review]

**Comment by Grosskopf** [comment]
what? i didn't want to close this :(

**Pull request closed without merging**